test/dhcp/server_isc_test.rb

```diff
@@ -82,8 +82,24 @@ def test_poap_quirks
   def test_loadSubnets_loads_managed_subnets
     subnets = @dhcp.loadSubnets
```

Comment:
In my understanding, `loadSubnets` implementation of `Proxy::DHCP::Server::ISC` load subnets only from config files and `Proxy::DHCP::SubnetService` is only here to store loaded data, so i don't see the problem to use `loadSubnets` here. For the other point, i've separated tests in several methods.